### PR TITLE
Update type definitions

### PR DESCRIPTION
**Pull request by mklucsarits**
https://wikia-inc.atlassian.net/browse/HG-587

### CODE REVIEW

File: tsd.json [hunk 1]
```diff
@@ -5,29 +5,98 @@
...
+    "fastclick/fastclick.d.ts": {
+      "commit": "7ca8add069411b3bde47fad7208706801cde8656"
+    },
+    "glob/glob.d.ts": {
+      "commit": "7ca8add069411b3bde47fad7208706801cde8656"
+    },
+    "google.analytics/ga.d.ts": {
+      "commit": "6e8f579f26c48e743f5490f6882fa1f0067ea855"
+    },
+    "gulp/gulp.d.ts": {
```
Comment (hakubo) [inline tsd.json]: since we have all gulp tasks written in JS,
what is the advantage of having those definitions in our repo?

Comment (mklucsarits) [inline tsd.json]: Ah you're right. Wasn't thinking of that.

File: typings/node/node.d.ts [hunk 2]
```diff
@@ -1,11 +1,11 @@
-// Type definitions for Node.js v0.10.1
```
Comment (rogatty) [inline typings/node/node.d.ts]: Is it a good time to update it? We're not using 0.12.0 yet.

**Comment by rogatty** [comment]
Check Jenkins or run tests locally to see errors related to type definitions.

**Pull request closed without merging**